Tools/Frame_params/SPARKKit-rover.param

```diff
@@ -0,0 +1,55 @@
+ACRO_TURN_RATE,90
+AHRS_ORIENTATION,14
+ATC_SPEED_I,0.7556667
+ATC_SPEED_P,0.7556667
+ATC_STR_ACC_MAX,160
+ATC_STR_RAT_D,0.002
+ATC_STR_RAT_FF,0.3316459
+ATC_STR_RAT_FLTD,2
+ATC_STR_RAT_FLTT,2
+ATC_STR_RAT_I,0.17
+ATC_STR_RAT_P,0.17
+ATC_TURN_MAX_G,0.15
+BATT_CURR_PIN,15
+BATT_VOLT_MULT,12.02
+BATT_VOLT_PIN,14
+CAN_P1_DRIVER,1
+COMPASS_USE2,0
+COMPASS_USE3,0
+CRUISE_SPEED,1.137718
+CRUISE_THROTTLE,85
+EK3_IMU_MASK,7
+FS_CRASH_CHECK,2
+FS_EKF_THRESH,0.6
+FS_GCS_TIMEOUT,2
+FS_THR_ENABLE,2
+GPS_POS1_X,-0.1
+GPS_POS1_Y,-0.08
+GPS_TYPE,9
+GUID_OPTIONS,64
+MANUAL_OPTIONS,1
+MANUAL_STR_EXPO,0.2
+MODE_CH,5
```

Comment:
The flight mode setting is user-specified and therefore unnecessary.

Reply:
@muramura Thank you for your suggestion. We plan to add copter kits in the near future and would like to have the same mode change switch for the convenience of our customers.